examples/storage/couchdb/couchdb-petset-service.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: couchdb
+  name: kubetest001
+  namespace: default
+spec:
+  #clusterIP: 100.10.10.101
+  ports:
+  - name: in-tcp-port
+    port: 80
+    targetPort: 5984
+  - name: rsync-port
+    port: 873
+    targetPort: 873
+  - name: ssh-port
+    port: 22
+    targetPort: 22
+  - name: erlang-port
+    port: 4369
+    targetPort: 4369
+  - name: foo-port
```

Comment:
Agree on the poor choice of name. The port itself is definitely needed, though - this is the port that the `vm.args` configures the Erlang VM to listen on for connection attempts from other Erlang nodes.